src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -5,17 +5,22 @@
 
     @if (!Model.HomeTeamId.HasValue && !Model.AwayTeamId.HasValue)
     {
+        string homeTeamName = Model.MustDisplayOnlyTeamNames ? Resources.UI.TournamentViews.HomeTeamPlaceholder 
```

Comment:
Why do you have this logic on UI?

Reply:
When I tried to use assigned values inside <strong> tag, view generated page with source code displayed instead of values.

So I decided to create variables in this view. And then it worked.